1-js/03-code-quality/06-polyfills/article.md

```diff
@@ -28,7 +24,7 @@ Actually, there are two parts in Babel:
     There's a term "polyfill" for scripts that "fill in" the gap and add missing implementations.
 
     Two interesting polyfills are:
-    - [babel polyfill](https://babeljs.io/docs/usage/polyfill/) that supports a lot, but is big.
```

Comment:
This is only true if you import the whole thing. I don't know where you put put this in this tutorial but you should mention https://github.com/babel/babel-preset-env/tree/v1.6.1 which shows how Babel can let you compile only the parts that you need to (including polyfills)

Reply:
Sure, could you elaborate that in the text a little bit, just as you said that here? 👍